WalletWasabi.Documentation/BackendDeployment.md

```diff
@@ -215,6 +215,15 @@ bitcoin-cli stop
 bitcoind
 ```
 
+### Upgrade Knots
+
+```sh
+bitcoin-cli stop
+sudo apt-get update && apt install --only-upgrade bitcoind
```

Comment:
```suggestion
sudo apt-get update && sudo apt install --only-upgrade bitcoind
```
?

Reply:
yes, needed, afaik.

Reply:
But are you sure that it'll update Knots via `apt`? I don't know if it is in a debian repo...

Afaik, this only works with something like @kixunil [cryptoanarchy debian repo](https://github.com/Kixunil/cryptoanarchy-deb-repo-builder/)...

Reply:
Well, it was working on the backend.

Reply:
ahh, I see, then yes it works, nice, I didn't know luke is hosting a debian repo.

Reply:
FYI add-apt-repository with ppa uses TOFU security model. Probably not too bad for coordinator but better avoid it in more critical pieces of SW.